tests/acceptance/blocks/filteroptionscomponent.js

```diff
@@ -0,0 +1,69 @@
+import { Selector, t } from 'testcafe';
+
+/**
+ * This class models user interactions with the {@link FilterOptionsComponent}.
+ */
+export default class FilterOptionsComponentBlock {
+  constructor (selector) {
+    this._container = '.yxt-FilterOptions-fieldSet';
+    this._selector = selector || Selector(this._container);
+  }
+
+  /**
+   * Returns whether this FilterOptions is collapsed or not.
+   */
+  async isCollapsed () {
+    return this._selector.find('.yxt-FilterOptions--collapsed').exists;
+  }
+
+  async getLabel (label) {
+    return this._selector.find('label').withText(label);
+  }
+
+  /**
+   * Return a selector for the option with the given label.
+   * @param {String} label
+   */
+  async getOption (label) {
```

Comment:
Minor nit, let's make this a private method (_getOption) since it's only used internally in this class. I think most consumers of this class will only want either the count or to toggle anyway.

Reply:
done